harvester_e2e_tests/integrations/test_upgrade.py

```diff
@@ -482,9 +482,8 @@ def test_version_compatibility(
         api_client.upgrades.delete(data['metadata']['name'])
         api_client.versions.delete(version)
 
-    def test_degraded_volume(
-        self, api_client, wait_timeout, vm_shell_from_host, vm_checker, upgrade_target, stopped_vm
-    ):
+    def test_degraded_volume(self, api_client, cluster_state, wait_timeout, vm_shell_from_host,
```

Comment:
it looks we forgot to remove `cluster_state` in parameters.

Reply:
Removed in 6a19ea8